File: JAVA/Daily Flash Codes/Week1/Day6/code_Java/prog2.java
/*
 *
 Program 2 : Write a Program to print Sum of First 10 Natural Numbers.
Output: The s um of First 10 Natural Numbers : 55
*/

import java.io.*;

class Program {
	public static void main(String[] args) throws IOException {
		int sum = 0;
	
		for(int i=1 ;i <= 10 ;i++)
			sum += i;

		System.out.println("Sum of first 10 natural num: "+ sum);

	}
}

File: C++/Daily Flash/week1/Day6/code_C++/prog2.cpp
/*
 * Program 2 : Write a Program to print Sum of First 10 Natural Numbers.
Output: The s um of First 10 Natural Numbers : 55
*
*/
#include <iostream>

int main() {

	
	int sum = 0;
	
	for(int i=1 ;i <= 10 ;i++)
		sum += i;

	std::cout << "Sum of first 10 integers = "<< sum  << std::endl;
	return 0;
}
